app-typescript/components/TreeMenu.tsx

```diff
@@ -428,13 +427,16 @@ export class TreeMenu<T> extends React.Component<IProps<T>, IState<T>> {
                                                 key={i}
                                                 option={option}
                                                 handleTree={this.handleTree}
-                                                onClick={onSelect(option)}
-                                                disabledItem={disabledItem(option)}
-                                                getBorderColor={this.props.getBorderColor}
-                                                getBackgroundColor={this.props.getBackgroundColor}
                                                 getId={this.props.getId}
                                                 optionTemplate={this.props.optionTemplate}
                                                 getLabel={this.props.getLabel}
+                                                disabledItem={disabledItem(option)}
```

Comment:
is there a reason you changed the order of props?

Reply:
actually nope...